lib/child-writer/writer-proxy.js

```diff
@@ -135,44 +79,32 @@ return permissions.isElevated().then((elevated) => {
           .invokeMap('replace', path.join(process.env.APPDIR, 'usr/'), '')
           .value();
 
-        return commandPrefix
-          .concat([ process.env.APPIMAGE ])
-          .concat(translatedArguments);
+        return _.concat([ process.env.APPIMAGE ], translatedArguments);
       }
 
-      return commandPrefix.concat(process.argv);
+      return process.argv;
     });
 
-    const command = commandJoin(commandArguments);
-
     // For debugging purposes
-    console.log(`Running: ${command}`);
-
-    return sudoPrompt.execAsync(command, {
-      name: packageJSON.displayName
-    }).then((stdout, stderr) => {
-      if (!_.isEmpty(stderr)) {
-        throw errors.createError({
-          title: stderr
-        });
-      }
+    console.log(`Running: ${commandArguments.join(' ')}`);
```

Comment:
Can we do it once we get analytics support outside the main Electron process, given that will take care of not printing the lines during tests, etc?